curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/how-do-you-like.md

```diff
@@ -0,0 +1,27 @@
+---
+id: 65529784e7cb6e24eb03a1af
+title: How Do You Like
+challengeType: 22
+dashedName: how-do-you-like
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+When you want to ask someone for their opinion or feelings about something, especially a place or an experience, you can use the phrase `How do you like`.
+
+Maria is asking Tom about his feelings or impressions of California.
```

Comment:
```suggestion
Maria is asking Tom about his impressions of California.
```
You can't really say "feeelings of", so I think it's fine to just drop that bit.